website/content/api-docs/system/internal-ui-default-auth-methods.mdx

```diff
@@ -0,0 +1,52 @@
+---
+layout: api
+page_title: /sys/internal/ui/default-auth-methods - HTTP API
+description: The '/sys/internal/ui/default-auth-methods' endpoint returns any configured or inherited default and backup auth methods for a given namespace.
+---
+
+# `/sys/internal/ui/default-auth-methods`
+
+@include 'alerts/enterprise-only.mdx'
+
+The `/sys/internal/ui/default-auth-methods` endpoint is used to return explicitly set or inherited default and backup auth methods for a given namespace.
```

Comment:
I wonder if we should add `types` so people don't mix this up with the endpoint that returns mount method data

```suggestion
The `/sys/internal/ui/default-auth-methods` endpoint is used to return explicitly set or inherited default and backup auth method types for a given namespace.
```